### DISCUSSION TITLE
📚 Documentation:  Where is the Appwrite Console?

### DISCUSSION THREAD

**Discussion by Abolfazl-MI** [appwrite/appwrite#4045] [Q&A] [answered]
### 💭 Description

in your documentation said how to run Docker and then where should I find way to go to consle !!!!!!!!!!!!!!!!!!!11 where is console ?
 

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by locohost** [comment, marked as answer]
Do you mean: After installing Appwrite, open browser, go to http://(your domain):(your port)?

**Comment by Ritzy-NCode** [comment]
I have recently set up appwrite on my local and I was able to access the console at the domain and port that I mentioned during installation. 
Are you looking for a specific instruction that says "go to the domain and port set up during installation", if yes, then that is not explicitly mentioned in the document.
